phoenix-core/src/it/java/org/apache/phoenix/end2end/ConnectionQueryServicesTestImpl.java

```diff
@@ -69,12 +69,14 @@ public ConnectionQueryServicesTestImpl(QueryServices services, ConnectionInfo in
     
     @Override
     public void addConnection(PhoenixConnection connection) throws SQLException {
+        super.addConnection(connection);
```

Comment:
Was this added intentionally? Do see a similar statement below?

Reply:
Yes, We are testing connection counts for different scenarios in LoggingConnectionLimiterIT, and these counts gets updated when we call acquire connection in CQSI's addConnection, which was not being used before as part of CQSTestImpl.

Reply:
But there is already a super.addConnection(..) below?

Reply:
Aaah I see....... ✌️